.github/workflows/scripts/weekly-label-check.js

```diff
@@ -0,0 +1,107 @@
+const { Octokit } = require("@octokit/rest");
+
+// Create Octokit instance
+const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+
+const REQUIRED_LABELS = ["priority", "role", "feature", "size"];
+const LABEL_MISSING = [
+  "priority: missing",
+  "role: missing",
+  "feature: missing",
+  "size: missing",
+];
```

Comment:
lets also add `issue level` as a required label and `issue level: missing` as a missing label. It's not technically part of the audit, but lets encourage people to use them. 


Secondly for scalability, can we create the `LABEL_MISSING` array dynamically from the `REQUIRED_LABELS` array? They all follow the `[REQUIRED_LABELS]: missing` pattern.